src/wrapmime.rs

```diff
@@ -175,21 +175,21 @@ pub fn mailimf_find_field(
 }
 
 /*the result is a pointer to mime, must not be freed*/
-pub unsafe fn mailmime_find_mailimf_fields(mime: *mut Mailmime) -> *mut mailimf_fields {
+pub fn mailmime_find_mailimf_fields(mime: *mut Mailmime) -> *mut mailimf_fields {
```

Comment:
Could this be replaced with `&mut Mailmime`? Safe function accepting raw pointer is strange

Reply:
+1 to this, safe functions accepting raw pointers is misleading